### PR TITLE
logout button in drawer

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Ref: #895 

This is only enabled in custom build.

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
code looks code, the icon has the wrong grey coloring.

@tobiasKaminsky what happens if I logout having multiple accounts configured? Will one get auto-chosen?

**Comment by tobiasKaminsky** [comment]
What is the correct color? 
I used #757575 which is the same as ic_user.png
ic_uploads.png is #6f6f6f...

**Comment by tobiasKaminsky** [comment]
> @tobiasKaminsky what happens if I logout having multiple accounts configured? Will one get auto-chosen?

The next in the list will be used.
But this is a customer request and there the app will be limited to one account. Then the login screen will be shown.

**Comment by AndyScherzinger** [comment]
👍 I fixed the coloring. 
@mario please review & merge :)

**Comment by mario** [comment]
@AndyScherzinger have you seen my comment above? I need that fixed before I'll want to merge.

**Comment by AndyScherzinger** [comment]
@mario which comment? I can't find any 0_o ?!

**Comment by mario** [comment]
@AndyScherzinger now you do xD

File: src/main/res/values/setup.xml [hunk 1]
```diff
@@ -38,6 +38,7 @@
     <bool name = "share_with_users_feature">true</bool>
     <bool name="show_whats_new">true</bool>
     <bool name="show_external_links">true</bool>
+    <bool name="logout">false</bool>
```
Comment (AndyScherzinger) [inline src/main/res/values/setup.xml]: fixed via https://github.com/nextcloud/android/pull/898/commits/d0521c1c894b18738d56db63fcf390e3c4a170c4

**Comment by AndyScherzinger** [comment]
👍 @mario I the liberty to fix the issue you mentioned via https://github.com/nextcloud/android/pull/898/commits/d0521c1c894b18738d56db63fcf390e3c4a170c4 :)

**Comment by mario** [comment]
👍  Thanks.

**Pull request merged**